client/http-client/Cargo.toml

```
@@ -14,6 +14,7 @@ async-trait = "0.1"
 rustc-hash = "1"
 hyper = { version = "0.14.10", features = ["client", "http1", "http2", "tcp"] }
 hyper-rustls = { version = "0.23", optional = true }
+hyper-proxy = { version = "0", default-features = false, features = ["rustls"] }
```

it should be possible to disable tls and not bring all these deps

```suggestion
hyper-proxy = { version = "0.9", default-features = false }
```